template.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jonathan work</title>
+    <link href="minimal.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+        <form action="" method="get">
+            <h2><b>Payment Form</b></h2>
+            Required fields are followed with*
+    <p>
```

Comment:
you dont need p tag here. First of all h2 is block tag by itself so I dont see why we need paragraph tag here. Maybe you wanted tu put Required fields are followed with*  into p tag ?